Jesus.py

```diff
@@ -0,0 +1,43 @@
+import pygame
+import sys
+import time
+from pygame.locals import *
+
+pygame.init()
+
+Width = 400
+Height = 585
+
+BLACK = (0, 0, 0)
+
+screen = pygame.display.set_mode((400, 585))
+image = pygame.image.load('Jesus.jpg')
+
+a = 100
+b = 200
+
+sprite = pygame.sprite.Sprite()

```

Comment:
Could leave making image into a sprite and do screen.blit( image, (0, 0)) and screen.blit(text, (0, 0) for example.